validate_json.js

```diff
@@ -1,71 +1,73 @@
- /*jshint esversion: 6 */
- const path = require('path');
- const fs = require('fs-extra')
-
- const PUBLIC = `public`
- const pathOfPublic = path.join(__dirname, PUBLIC);
-
+/*jshint esversion: 6 */
+// Requirements
+const path = require('path')
+const fs = require('fs-extra')
+
+const PUBLIC = 'public'
+const pathOfPublic = path.join(__dirname, PUBLIC)
+const VERSION = 2
+const LOGO_EXTENSION = '.png'
+const ONE_APP_EXTENSION = '.json'
+const APP_FOLDER = 'apps'
+const LOGO_FOLDER = 'logos'
+
+// VALIDATION functions
+const isLogoValid = ({ existsSync, statSync }, logoFullPath) => existsSync(logoFullPath) && statSync(logoFullPath).isFile()
+const isVersionValid = (applicationVersion, capVersion) => applicationVersion === capVersion
+
+// Displayed Messages
+const printablePath = fullPath => fullPath.substr(fullPath.indexOf(`/${PUBLIC}`))
+const versionErrorMessage = (appVersion, capVersion, filename) => `unmatched versions ${appVersion} ${capVersion} for ${filename}`
+const logoErrorMessage = (filename, logoFullPath) =>`Cannot find logo for ${filename} ${printablePath(logoFullPath)}`
+const defaultDisplayName = appName => appName.charAt(0).toUpperCase() + appName.slice(1)
+const endMessage = appNumber => `${appNumber} applications validated.`
+const validatedMessage = filename => `Validated ${filename}`
```

Comment:
Two notes here:
- Some of these method names are variable names, method names usually have a verb. 
- Extraction to a method is good if it's repeating itself. Most of these methods (if not all of them) are only used once. Inlining them will make the code much easier to read.